MTFO/Patches/Patch_DiscordManager.cs

```diff
@@ -0,0 +1,23 @@
+using MTFO.Utilities;
+using HarmonyLib;
+
+namespace MTFO.Patches
+{
+    [HarmonyPatch(typeof(DiscordManager), nameof(DiscordManager.UpdateDiscordDetails))]
+    static class Patch_DiscordManager
+    {
+        [HarmonyPrefix]
+        private static bool Pre_UpdateDiscordDetails(eDiscordDetailsDisplay details)
+        {
+            if (details == null) return true;
+            if (Globals.Global.RundownIdToLoad != 1) return true;
+            String detailName = details.ToString();
+            if (detailName.Equals("OnSuccess") || detailName.Equals("OnFail"))
```

Comment:
Why are you converting the enum to a string and checking the string value? Just compare the enum value itself.
```suggestion
            if (details == eDiscordDetailsDisplay.OnSuccess || details == eDiscordDetailsDisplay.OnFail)
```